### ISSUE TITLE
Set up sponsorship

### ISSUE THREAD

**Issue by djmitche** [freecinc/freecinc-issues#14]
I'm midway through this -- just need to figure out the financial bits.

**Comment by djmitche** [comment]
It appears that I can only allow sponsorship of the freecinc org if I have a business account.  Instead, I've applied for personal sponsorship support.  Once that's approved, I'll put a link on the freecinc homepage and the org READMEs.

**Comment by djmitche** [comment]
There seems to be some snag in the GitHub machinery.. soon.

**Comment by djmitche** [comment]
Hey presto:

![image](https://user-images.githubusercontent.com/28673/146477198-a3ad4e2b-7ce3-436a-b715-65bec0ee72e3.png)

**Comment by jaker-dotcom** [comment]
I appears to me that sponsoring is no longer possible.

**Comment by djmitche** [comment]
I removed support.  I had one sponsor, but then Stripe wanted a bunch of paperwork that was substantially more trouble than the reward of the sponsorship, so I removed it.

The issues with the service right now are not financial.  Rather, a k8s upgrade on Digitalocean destroyed the existing cluster, and now that I've rebuilt it in GKE, using Google load balancers, taskd seems to be crashing much more frequently.

Fixes:
 * Find somewhere / someone else to host this where it is not in a dedicated cluster and where it is admin'd by someone with more k8s experience than I have.  I'm happy to share this responsibility.
 * Replace the sync backend with Taskchampion (long-term, but in progress)
